python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -160,6 +191,44 @@ def convert_weights_and_inputs(node, **kwargs):
         return [tval_node]
 
 
+@mx_op.register("Deconvolution")
+def convert_deconvolution(node, **kwargs):
+    helper, _, _ = import_onnx_modules()
+    name = node["name"]
+    inputs = node["inputs"]
+
+    num_inputs = len(inputs)
+
+    proc_nodes = kwargs["proc_nodes"]
+    input_node = proc_nodes[kwargs["index_lookup"][inputs[0][0]]].name
+    weights_node = proc_nodes[kwargs["index_lookup"][inputs[1][0]]].name
+
+    if num_inputs > 2:
+        bias_node = proc_nodes[kwargs["index_lookup"][inputs[2][0]]].name
+
+    attrs = node.get("attrs")
+    num_filter = int(attrs["num_filter"])
+    kernel_dims = list(parse_helper(attrs, "kernel"))
+    stride_dims = list(parse_helper(attrs, "stride", [1, 1]))
+    pad_dims = parse_padding(attrs)
+    num_group = int(attrs.get("num_group", 1))
+    input_nodes = [input_node, weights_node]
+    if num_inputs > 2:
+        input_nodes.append(bias_node)
+
+    deconv_node = helper.make_node(
```

Comment:
convtranspose in onnx also takes dilations, output_shape, output_padding attributes. I think we should provide those values as well.